### ISSUE TITLE
Any way I can help?

### ISSUE THREAD

**Issue by Lumexralph** [EventStore/EventStore-Client-Go#34]
Hi,

My company is currently looking at eventstore to add event sourcing to our system, I am currently prototyping with one of our microservice and we are a Go shop team.

Is there any plan on making the release of a stable client soon? Is there any way I can help?

**Comment by hayley-jean** [comment]
Hi @Lumexralph!

We intend on paying more attention to the go client starting February, and will aim to bring it up to the same level as the other supported gRPC clients.

If you want to help out in the meantime, there are a few issues in this repo already about functionality that's currently missing. The dotnet gRPC client is the model client that other clients should reference when being written.

If you need any more details on any of the issues, feel free to ask in the issue and we'll do our best to clarify

**Comment by Lumexralph** [comment]
Hi @hayley-jean,

I think the first issue I had was when I wanted to connect to an event store DB cluster, I had an error from the Go HTTP client, about `localhost not a recognized protocol`. I had to adjust the `gossip seeds/URL` in my code in the `Configuration` struct, to include `https://localhost:2113`, that was how I got it working.

I will be glad to make a PR but I don't know `.NET`, probably having some issues up that needs help and I will be glad to give my input. 

I am also glad the library will resume active development next month.
